### ISSUE TITLE
want to ask some details about the paper

### ISSUE THREAD

**Issue by HiBugs** [kaiwang960112/Self-Cure-Network#5]
Because you haven't released the code yet, I want to ask some questions. 
1. I see that the RAF-DB dataset only contains the training set and test set, did you use the test set as validation set when training? 
2. The image size of some datasets is far less than 224*224, why didn't use the origin size? 
3. Did you use the data augmentation when training?
Thanks.

**Comment by kaiwang960112** [comment]
1. We use all the train set as training data and all the test set as testing data.
2. The original size is 100*100, we resize to 224*224 in our paper. The size doesn't impact much, you can have a try.
3. We don't use data augmentation in the training stage.
Thank you! We will release the code recently.
